aiter/ops/triton/mha.py

```diff
@@ -3207,6 +3328,8 @@ def flash_attn_varlen_func(
     return_lse=False,
     return_attn_probs=False,
     block_table=None,
+    fused_backward=False,
+    onekernel_backward=False,
```

Comment:
Remove these args from the top level API. Instead add a setter function which sets a global variable to control which kernel to use. See the MoE code for example https://github.com/ROCm/aiter/blob/main/aiter/ops/triton/moe_op.py#L18